governance.md

```diff
@@ -0,0 +1,69 @@
+# Brigade Governance
+
+The following document describes how the Brigade project governance operates.
+
+## The Brigade Project
+
+The Brigade project is made up of several codebases with different release cycles. These codebases include:
+
+- Brigade
+- Kashti
+- gateways, libraries, and integrations
+
+There are two levels of maintainers for Brigade: org maintainers and project maintainers.
```

Comment:
continue from Helm: "The Brigade org maintainers oversee the overall project and its health. Project maintainers focus on a single codebase or a group of related codebases. For example, a gateway maintainer manages a gateway repository or a utility project has a related project manager."

Reply:
I agree on that one.